package.json

```diff
@@ -32,26 +32,29 @@
     "@storybook/addon-links": "5.0.11",
     "@storybook/addon-storyshots": "5.0.11",
     "@storybook/react": "5.0.11",
+    "@testing-library/jest-dom": "^5.5.0",
+    "@testing-library/react": "^10.0.2",
+    "@testing-library/user-event": "^10.0.2",
     "@types/bn.js": "^4.11.4",
     "@types/jest": "^20.0.2",
     "@types/memoize-one": "^4.1.1",
     "@types/ramda": "^0.25.50",
     "@types/react": "16.8.6",
     "@types/react-dom": "16.0.6",
     "@types/react-spinkit": "^3.0.5",
-    "@types/react-test-renderer": "16.0.1",
     "@types/storybook__react": "4.0.1",
     "awesome-typescript-loader": "^5.2.1",
     "concurrently": "^4.1.0",
     "graphql-language-service": "^2.0.0",
     "graphql-syntax-highlighter-react": "^0.3.5",
     "husky": "^4.2.5",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.9.0",
+    "ignore-loader": "^0.1.2",
+    "jest": "25.0.0",
     "prettier": "^2.0.4",
     "pretty-quick": "^2.0.1",
     "react": "^16.8.6",
-    "react-dom": "16.8.2",
+    "react-dom": "^16.9.0",
```

Comment:
We updated react dom because we were getting this warning with the new test library: 
```
   It looks like you're using a version of react-dom that supports the "act" function, but not an awaitable version of "act" which you will need. Please upgrade to at least react-dom@16.9.0 to remo
ve this warning.
```